collect_app/src/main/java/org/odk/collect/android/utilities/PermissionUtils.java

```diff
@@ -41,322 +42,287 @@
 
 public class PermissionUtils {
 
-    private PermissionUtils() {
+    /**
+     * Required for context and spawning of Dexter's activity that handles
+     * permission checking.
+     */
+    private final Activity activity;
+
+    public PermissionUtils(@NonNull Activity activity) {
+        this.activity = activity;
+    }
+
+    public static boolean isStoragePermissionGranted(Context context) {
+        return isPermissionGranted(context,
+                Manifest.permission.READ_EXTERNAL_STORAGE,
+                Manifest.permission.WRITE_EXTERNAL_STORAGE);
+    }
+
+    public static boolean isCameraPermissionGranted(Context context) {
+        return isPermissionGranted(context, Manifest.permission.CAMERA);
+    }
+
+    public static boolean isLocationPermissionGranted(Context context) {
```

Comment:
Should be a plural form `isLocationPermissionsGranted` if below you use a plural form for `isCameraAndRecordAudioPermissionsGranted`.